src/core/dbus.c

```diff
@@ -860,6 +860,10 @@ int bus_init_api(Manager *m) {
         if (r < 0)
                 return log_error_errno(r, "Failed to set up API bus: %m");
 
+        r = bus_track_coldplug(bus, &m->subscribed, /* recursive= */ false, m->deserialized_subscribed);
```

Comment:
bus_track_coldplug will fail if any of the provided names disappeared in the mean time. i think we should just ignore failures here, i.e. cast the result to (void)